recipe/conda_build_config.yaml

```diff
@@ -8,66 +8,53 @@ docker_image:                                # [os.environ.get("BUILD_PLATFORM",
 
    - quay.io/condaforge/linux-anvil-cuda:9.2        # [linux64 and (os.environ.get("BUILD_PLATFORM") == "linux-64" or os.environ.get("CONFIG_VERSION", "1") == "1")]
    - quay.io/condaforge/linux-anvil-cuda:10.0       # [linux64 and (os.environ.get("BUILD_PLATFORM") == "linux-64" or os.environ.get("CONFIG_VERSION", "1") == "1")]
-   - quay.io/condaforge/linux-anvil-cuda:10.1       # [linux64 and (os.environ.get("BUILD_PLATFORM") == "linux-64" or os.environ.get("CONFIG_VERSION", "1") == "1")]
-   - quay.io/condaforge/linux-anvil-cuda:10.2       # [linux64 and (os.environ.get("BUILD_PLATFORM") == "linux-64" or os.environ.get("CONFIG_VERSION", "1") == "1")]
+   - quay.io/condaforge/linux-anvil-cos7-cuda:10.1  # [linux64 and (os.environ.get("BUILD_PLATFORM") == "linux-64" or os.environ.get("CONFIG_VERSION", "1") == "1")]
+   - quay.io/condaforge/linux-anvil-cos7-cuda:10.2  # [linux64 and (os.environ.get("BUILD_PLATFORM") == "linux-64" or os.environ.get("CONFIG_VERSION", "1") == "1")]
    - quay.io/condaforge/linux-anvil-cuda:11.0       # [linux64 and (os.environ.get("BUILD_PLATFORM") == "linux-64" or os.environ.get("CONFIG_VERSION", "1") == "1")]
    - quay.io/condaforge/linux-anvil-cuda:11.1       # [linux64 and (os.environ.get("BUILD_PLATFORM") == "linux-64" or os.environ.get("CONFIG_VERSION", "1") == "1")]
    - quay.io/condaforge/linux-anvil-cuda:11.2       # [linux64 and (os.environ.get("BUILD_PLATFORM") == "linux-64" or os.environ.get("CONFIG_VERSION", "1") == "1")]
 
-   # these two variants are needed for cuTENSOR support
-   - quay.io/condaforge/linux-anvil-cos7-cuda:10.1  # [linux64 and (os.environ.get("BUILD_PLATFORM") == "linux-64" or os.environ.get("CONFIG_VERSION", "1") == "1")]
-   - quay.io/condaforge/linux-anvil-cos7-cuda:10.2  # [linux64 and (os.environ.get("BUILD_PLATFORM") == "linux-64" or os.environ.get("CONFIG_VERSION", "1") == "1")]
-
 c_compiler_version:     # [linux]
   - 7                   # [linux64 or aarch64]
   - 8                   # [ppc64le]
 cxx_compiler_version:   # [linux]
   - 7                   # [linux64 or aarch64]
   - 8                   # [ppc64le]
 
-cuda_compiler:   # [linux64]
-  - nvcc         # [linux64]
+cuda_compiler:   # [linux64 or win]
+  - nvcc         # [linux64 or win]
 
 cuda_compiler_version:
-  - None
+  - None                       # [linux64]
   - None                       # [linux64]
   - 9.2                        # [linux64]
   - 10.0                       # [linux64]
   - 10.1                       # [linux64]
```

Comment:
Did we want these to have `or win`?

Reply:
I was messing around locally with the v8.6 PR (so you can see I pushed a wrong setup). There's no way to get this done correctly for Windows + CUDA 10.0/10.1, so I give up. We can get things rendered, but for some reason there appears a big matrix of two cuDNN versions times all CUDA versions (times Python), which is wrong.